server/BREAKING.md

```diff
@@ -1,5 +1,10 @@
 > **Note:** These breaking changes are only relevant to the server packages and images released from `./routerlicious`.
 
+## 0.1039 Breaking Changes
```

Comment:
@tylerbutler should I bump the package.json version number in server to cause the bump in version for server? Or does that happen automatically on merging into next?

Reply:
server changes should be separate from protocol-defs changes, so split them out.

server changes must ONLY be made in main, and every release of server is a major release. That means to consume a server change in client you have to make the server change in main, prerelease it, then integrate it into client _on next_. @NicholasCouri is going through this dance now, in case he has a better way to explain it.